benchmark/gaia/gaia_benchmark.py

```diff
@@ -0,0 +1,203 @@
+import os, json, re, string, warnings
+from camel.agents import ChatAgent
+from typing import Literal, List, Union
+from camel.messages import BaseMessage
+import time
+SCRIPT_PATH = os.path.realpath(__file__)
+SCRIPT_DIR = os.path.dirname(SCRIPT_PATH)
+DATASET_DIR = os.path.join(SCRIPT_DIR, "Dataset")
+if not os.path.isdir(DATASET_DIR):
+    os.mkdir(DATASET_DIR)
+class GAIABenchmark:
+    r"""
+    Please using huggingface-cli login to get authorization
+    From Hugging Face download GAIA dataset
+    This Class will create a folder to cache GAIA dataset
+    """
+    def __init__(self) -> None:
+        self.validation_tasks = [[], [], []]
+        self.test_tasks = [[], [], []]
```

Comment:
I normally don't like hardcoded, I think you could use a dict to dynamically figure out what are the task levels. So that we don't need the hardcoded numbers(1,2,3)